fl4health/clients/basic_client.py

```diff
@@ -701,8 +701,8 @@ def validate(self) -> Tuple[float, Dict[str, Scalar]]:
             )
             # There will be no clashes due to the naming convention associated with the metric managers
             if self.num_test_samples is not None:
-                val_metrics["test - num_examples"] = int(self.num_test_samples)
-            val_metrics["test - loss"] = test_loss
+                val_metrics[TestMetricPrefix.TEST_PREFIX.value + "num_examples"] = self.num_test_samples
```

Comment:
I'm good with us defining the constants
```
TEST_NUM_EXAMPLES = f"{TestMetricPrefix.TEST_PREFIX.value} num_examples"
TEST_LOSS = f"{TestMetricPrefix.TEST_PREFIX.value} loss"
```
In the same file that you have the enum and just importing them so you don't have to repeat this structure everywhere.

Reply:
done!